src/expr.ts

```diff
@@ -448,6 +448,10 @@ const fnMapDatasetToValue: FnMapDatasetToValue = {
         // return number
         return d3.median(dataset.getValues());
     },
+    percentile: function (dataset, percentage, renderInfo) {
```

Comment:
This won't work since expression functions can only accept `dataset` and `renderInfo`. I checked locally and it won't even compile when I run `npm run dev`. I would suggest instead looking at the [RenderInfo class](https://github.com/pyrochlore/obsidian-tracker/blob/master/src/data.ts#L555-L650) and adding a new field there to hold `percentage`. Then your dataView code would look something like:

```
searchType: dvField
searchTarget: dataviewTarget
folder: /diary
endDate: 2021-01-03
percentage: 0.6
summary:
    template: '60-percentile value: {{percentage(dataset(0))::.1f}} <-- should be 29.6'
```

Reply:
After you have made changes, I strongly encourage you to run `npm run dev` and then open up the example folder in Obsidian so you can verify it works (this is what I do with every new feature submitted)